css-three-column-layout/styles.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  width: 1400px;
+  height: 600px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+}
+
+.container-2 {
+  width: 1000px;
+  margin: 0 auto;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  width: 100%;
+  height: 100px;
```

Comment:
Shouldn't be giving this a `height`. You want the content to dictate the `height`